File: Challenge 1/Java/Program.java
public class Program {

  static String[] TESTS = {
    "HELLO WORLD",
    "SOMETHING ELSE",
    "ABCDEFG"
  };

  public static void main(String[] args) {
    for (String test : TESTS) {
      Character result = FindRepeatCharacter(test);
      System.out.println(
        String.format("%s : %s", test, result)
      );
    }
  }

  public static Character FindRepeatCharacter(String input) {
    return null;
  }
}

File: Challenge 1/csharp/Program.cs
using System;

namespace dotnet
{
    class Program
    {
        static String[] TESTS = 
        {
            "HELLO WORLD",
            "SOMETHING ELSE",
            "ABCDEFG"
        };

        static void Main(string[] args)
        {
            foreach (var test in TESTS) 
            {
                var result = FindRepeatCharacter(test);
                Console.WriteLine("{0} : {1}", test, result);
            }
        }

        static char? FindRepeatCharacter(String input) {
            return null;
        }
    }
}
